src/renderer/dx/CustomTextLayout.cpp

```diff
@@ -133,13 +133,8 @@ CustomTextLayout::CustomTextLayout(IDWriteFactory1* const factory,
         RETURN_IF_FAILED(_analyzer->AnalyzeBidi(this, 0, textLength, this));
         RETURN_IF_FAILED(_analyzer->AnalyzeScript(this, 0, textLength, this));
         RETURN_IF_FAILED(_analyzer->AnalyzeNumberSubstitution(this, 0, textLength, this));
-
-        // Perform our custom font fallback analyzer that mimics the pattern of the real analyzers.
-        // Fallback routines are not available below Windows 8.1, so just skip them and let a replacement character happen.
-        if (IsWindows8Point1OrGreater())
-        {
-            RETURN_IF_FAILED(_AnalyzeFontFallback(this, 0, textLength));
-        }
+        //

```

Comment:
I believe we still need the comment stating...

```suggestion
        // Perform our custom font fallback analyzer that mimics the pattern of the real analyzers.
```